website/meta/universe.json

```
@@ -1715,7 +1715,36 @@
                 "\tprint(ent.text, ent.start_char, ent.end_char, ent.label_)"
             ],
             "author": "Stefan Daniel Dumitrescu, Andrei-Marius Avram"
-        }
+        },
+        {
+            "id": "num_fh",
+            "title": "Numeric Fused-Head",
+            "slogan": "Numeric Fused-Head Identificaiton and Resolution in English",
+            "description": "This package provide a wrapper for the Numeric Fused-Head in English. It provides another information layer information on numbers that refer to another entity which is not     obvious from the syntactic tree.",
+            "github": "yanaiela/num_fh",
+            "pip": "num_fh",
+            "category": ["pipeline", "models", "research"],
```

```suggestion
            "category": ["pipeline", "research"],
```

So far, we've mostly been using the `models` category for pre-trained model packages etc. So I'd suggest only using `pipeline` and `research` here 🙂